quizcon/main/models.py

```diff
@@ -16,7 +16,7 @@ class Quiz(models.Model):
     randomize = models.BooleanField(
         default=False,
         help_text="Randomize the quiz questions")
-
+    scoring_scheme = models.IntegerField(default=1)
```

Comment:
As the scoring_scheme field captures a set of known values, adding a `choices` attribute is a best practice here. See here for the Django documentation: https://docs.djangoproject.com/en/3.2/ref/models/fields/#choices Defining these known values also makes the code more sustainable and avoids "magic numbers". (https://en.wikipedia.org/wiki/Magic_number_(programming))

Reply:
@sdreher Yes I'll make those changes!